exercises/nth-prime/nth_prime_test.rb

```diff
@@ -22,13 +25,35 @@ def test_sixth_prime
 
   def test_big_prime
     skip
-    assert_equal 104_743, Prime.nth(10_001)
+    assert_equal 104743, Prime.nth(10001)
   end
 
-  def test_weird_case
+  def test_there_is_no_zeroth_prime
     skip
     assert_raises ArgumentError do
       Prime.nth(0)
     end
   end
```

Comment:
I updated [example.tt](https://github.com/cacqw7/xruby/blob/9b1b74bf000466805dd6d2559fd928802a084a9b/exercises/nth-prime/example.tt#L13) to generate the exception test.